### PR TITLE
[16.0][ADD] web_widget_remaining_days_exact_date: New module to show exact date when remaining days shown

### PR DESCRIPTION
**Pull request by CarlosRoca13**
This module is adding an option to the remaining_days widget to be able to show the exact date.
image

cc @Tecnativa TT50517

ping @pedrobaeza @chienandalu

### CODE REVIEW

File: web_widget_remaining_days_exact_date/static/src/remaining_days/remaining_days.esm.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import {patch} from "@web/core/utils/patch";
+import {sprintf} from "@web/core/utils/strings";
+
+patch(
+    RemainingDaysField.prototype,
+    "web_widget_remaining_days_exact_date.RemainingDaysField",
+    {
+        get diffString() {
+            var value = this._super(...arguments);
+            if (this.props.exact_date && Math.abs(this.diffDays) <= 99) {
```
Comment (chienandalu) [inline web_widget_remaining_days_exact_date/static/src/remaining_days/remaining_days.esm.js]: Add comment on why the hardcoded 99 value :)

**Review by chienandalu — APPROVED** [review]
Tested. Tiny suggestion:

**Comment by CarlosRoca13** [comment]
I have changed the module's behavior so that this option works by default, and in case you don't want to use it, it can be disabled with options="{'exact_date': False}". Please review it again @chienandalu

**Review by chienandalu — APPROVED** [review]
:+1:

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**